File: book_shop/html/script/models/MainSlider.js
class MainSlider{
    #imageName;
    #toolTipText;
    #sliderId;
    #isActive;
    constroctor(sliderId, imageName, toolTipText, isActive)
    {
        this.#sliderId = sliderId;
        this.#imageName = imageName;
        this.#toolTipText = toolTipText;
        this.#isActive = isActive;
    }
    get sliderId()
    {
        return this.#sliderId;
    }
    get imageName()
    {
        return this.#imageName;
    }
    get toolTipText()
    {
        return this.#toolTipText;
    }
    get isActive()
    {
        return this.#isActive;
    }
    set sliderId(val)
    {
        this.#sliderId = val;
    }
    set imageName(val)
    {
        this.#imageName = val;
    }
    set toolTipText(val)
    {
        this.#toolTipText = val;
    }
    set isActive(val)
    {
        this.#isActive = val;
    }
    static from(json){
        return Object.assign(new MainSlider(), json);
    }
}
export  {MainSlider}

File: BookShop/BookShop/BookShopApp/Models/MainSlider.cs
using System.ComponentModel.DataAnnotations;

namespace BookShopApp.Models;

public class MainSlider
{
    [Key]
    public int SliderId { get; set; }
    public string ImageName { get; set; }
    public string ToolTipText { get; set; }
    public bool IsActive {get; set;}
    public MainSlider(string imageName, string tooltipText)
    {
        this.ImageName = imageName;
        this.ToolTipText = tooltipText;
    }
    public MainSlider(int sliderId, string imageName, string tooltipText, bool isActive)
    {
        this.SliderId = sliderId;
        this.ImageName = imageName;
        this.ToolTipText = tooltipText;
        this.IsActive = isActive;
    }
}
